src/Master.cxx

```diff
@@ -23,7 +23,15 @@ void Master::handle_command_message(std::unique_ptr<KafkaW::Msg> &&msg) {
   command_handler.handle({(char *)msg->data(), (int32_t)msg->size()});
 }
 
+void Master::handle_command(rapidjson::Document &cmd) {
+  CommandHandler command_handler(this, config.config_file);
+  command_handler.handle(cmd);
+}
+
 void Master::run() {
+  for (auto &cmd : config.commands_from_config_file) {
```

Comment:
Could this be `auto const &cmd`? It would require a few method signatures to change, including `handle_command`.